File: WebContent/dwr/interface/ScheduleService.js
var ScheduleService = {};
ScheduleService.dummyData = [
	{kl: '09.00', team1: 'Lag 1', team2: 'Lag 2'},
	{kl: '09.06', team1: 'Lag 3', team2: 'Lag 4'}
];
ScheduleService.createSchedule = function(startTime, teams, callback) {
	callback.callback(ScheduleService.dummyData);
};

File: src/no/fll/web/ScheduleService.java
package no.fll.web;

public class ScheduleService {

	public Schedule[] createSchedule(String startTime, int teams) {
		return new Schedule[]{new Schedule("09.00", "Lag 1", "Lag 2"), new Schedule("09.06", "Lag 3", "Lag 4")};
	}
}
